components/Overview.js

```diff
@@ -0,0 +1,35 @@
+import { Fragment } from "react";
+import Image from "next/image.js";
+import { styled } from "styled-components";
+import pictures from "../db.js";
+
+export default function Overview() {
+  return (
+    <ul>
+      {pictures.map((picture) => (
+        <li key={picture.id}>
+          <StyledImage
+            src={picture.image}
+            alt={picture.theme}
+            height={300}
+            width={200}
+          />
+
+          <StyledTheme>{picture.username}</StyledTheme>
+        </li>
+      ))}
+    </ul>
+  );
+}
+
+const StyledImage = styled(Image)`
+  object-fit: contain;
+  width: 100%;
```

Comment:
Since you defined with and height on StyledImage directly in line 14/15, you should not need this anymore

Reply:
Hey, thank you for reviewing my code. I first deleted these lines, as suggested. After that, the picture sizes changed and looked terrible. Maybe I can leave it as it is for now, and fix the styling later?

Reply:
Hey CJ,
I fixed a part of the image styling so it is more responsive. However, I don't understand what I should do with width and height applied directly to the Image element...
Can you explain what you mean a little more explizit?

Reply:
You don't need to do anything, you're all good. :) I just offered an explanation regarding the difference between the values.